### PR TITLE
Update contract verification page for etherscan and mainnet

### PR DESCRIPTION
**Pull request by isabellewei**
## Description

Add instructions for contract verification with Etherscan and mainnet

### CODE REVIEW

File: src/content/docs/en/developers/verifying-smart-contracts.mdx [hunk 1]
```diff
@@ -82,18 +85,54 @@ npx hardhat verify --network scrollSepolia 0xD9880690bd717189cC3Fbe7B9020F27fae7
...
+  --etherscan-api-key <your Scrollscan API key> \
+  --constructor-args <your constructor arguments>
+```
+<Aside type="caution" title="Caution">
+  Do not specify the chain ID.
+</Aside>
+
+#### Blockscout
+
+{/* <Aside type="danger" title="Warning">
```
Comment (isabellewei) [inline src/content/docs/en/developers/verifying-smart-contracts.mdx:112]: do we still want to include this here?

File: src/content/docs/en/developers/verifying-smart-contracts.mdx [hunk 2]
```diff
@@ -82,18 +85,54 @@ npx hardhat verify --network scrollSepolia 0xD9880690bd717189cC3Fbe7B9020F27fae7
...
   Our Blockscout instance currently has inconsistent behavior in verifying contracts with Foundry. We're working on a
   fix, along with other stability improvements. Don't hesitate to come ask for help [on
   Discord](https://discord.com/channels/853955156100907018/1028102371894624337) though.
-</Aside>
+</Aside> */}
```
Comment (isabellewei) [inline src/content/docs/en/developers/verifying-smart-contracts.mdx]: do we still want to include this here?

File: src/content/docs/en/developers/verifying-smart-contracts.mdx [hunk 3]
```diff
@@ -5,31 +5,39 @@ title: "Verifying Smart Contracts"
...
   deployments. As additional services come online for mainnet, we'll update these instructions.
-</Aside>
+</Aside> */}
 
 ## Using Developer Tools
 
-Most smart contract tooling has plugins for verifying your contracts easily on Etherscan. Blockscout supports Etherscan's contract verification APIs, and it's straightforward to use these tools with the Scroll Sepolia Testnet.
+Most smart contract tooling has plugins for verifying your contracts easily on Etherscan. Blockscout supports Etherscan's contract verification APIs, so it's straightforward to use these tools using the APIs of either of these block explorers.
+
+| Network    | Scroll Scroll                    | Sepolia                                  |
```
Comment (dghelm) [inline src/content/docs/en/developers/verifying-smart-contracts.mdx]: "Scroll | Scroll Sepolia"

File: src/content/docs/en/developers/verifying-smart-contracts.mdx [hunk 4]
```diff
@@ -5,31 +5,39 @@ title: "Verifying Smart Contracts"
...
+excerpt: "Easily verify your smart contracts with Scroll-supported developer tooling"
 ---
 
 import Aside from "../../../../components/Aside.astro"
 import ClickToZoom from "../../../../components/ClickToZoom.astro"
 import verify1 from "./_images/verify1.png"
 import CodeSample from "../../../../components/CodeSample/CodeSample.astro"
 
-After deploying your smart contracts, it's important to verify your code on [our block explorer](https://scrollscan.com/) or the [Sepolia block explorer](https://sepolia-blockscout.scroll.io).
+After deploying your smart contracts, it's important to verify your code on our block explorers. This can be done in an automated way using your developer tooling or the Web UI.
```
Comment (dghelm) [inline src/content/docs/en/developers/verifying-smart-contracts.mdx]: I'm wondering if "our" is right if we're not running Scrollscan and other blockexplorers might want visibility on this page.

"on a block explorer"

File: src/content/docs/en/developers/verifying-smart-contracts.mdx [hunk 5]
```diff
@@ -5,31 +5,39 @@ title: "Verifying Smart Contracts"
...
 
 ## Using Developer Tools
 
-Most smart contract tooling has plugins for verifying your contracts easily on Etherscan. Blockscout supports Etherscan's contract verification APIs, and it's straightforward to use these tools with the Scroll Sepolia Testnet.
+Most smart contract tooling has plugins for verifying your contracts easily on Etherscan. Blockscout supports Etherscan's contract verification APIs, so it's straightforward to use these tools using the APIs of either of these block explorers.
+
+| Network    | Scroll Scroll                    | Sepolia                                  |
+| ---------- | -------------------------------- | ---------------------------------------- |
+| Scrollscan | https://api.scrollscan.com/api   | https://api-sepolia.scrollscan.com/api   |
+| Blockscout | https://blockscout.scroll.io/api | https://sepolia-blockscout.scroll.io/api |
```
Comment (dghelm) [inline src/content/docs/en/developers/verifying-smart-contracts.mdx:25]: Do we even want to emphasize Blockscout? For the most part we consider it an internal tool and don't want developers defaulting to using it for contract verification.

Comment (dghelm) [reply src/content/docs/en/developers/verifying-smart-contracts.mdx:25]: Actually, for now let's leave it.

File: src/content/docs/en/developers/verifying-smart-contracts.mdx (hunk 2 above)
Comment (dghelm) [reply src/content/docs/en/developers/verifying-smart-contracts.mdx]: Probably not.

**Review by dghelm — DISMISSED** [review]
Thanks for getting this updated! One nit and a few thoughts, but I'll try to get it live ASAP.

**Pull request merged**